API/Entities/Category.cs

```diff
@@ -1,15 +1,30 @@
+
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
 namespace API.Entities
 {
     public class Category
     {
+        public int ParentCategoryId { get; set; }
+
+        [Required]
         public int Id { get; set; }
 
+        [Required]
+        [MaxLength(100)]
         public string Name { get; set; }
 
-        public int ParentCategoryId { get; set; }
-
+        [Required]
         public int UserId { get; set; }
 
+        [Required]
         public string OperationTypeId { get; set; }
```

Comment:
To niezly string. Przez to ci zrobilo dodatkowa kolumne w tej tabeli OperationTypeId1